postreise/analyze/generation/summarize.py

```diff
@@ -149,7 +149,7 @@ def _groupby_state(index: str) -> str:
 
 
 def get_generation_time_series_by_resources(scenario, area, resources, area_type=None):
-    """Get time series generation for each resource in certain area of a scenario
+    """Gett hourly total generation for generator type(s) in an area
```

Comment:
Gett -> Get

Reply:
Not sure when to have period when should not? Both in function descriptions and parameter descriptions.

Reply:
I usualyl add a period

Reply:
Done